elyra/kfp/operator.py

```diff
@@ -254,7 +254,12 @@ def __init__(self,
             run_name_placeholder = '{{workflow.annotations.pipelines.kubeflow.org/run_name}}'
             self.container.add_env_variable(V1EnvVar(name='ELYRA_RUN_NAME',
                                                      value=run_name_placeholder))
-        else:
+        elif workflow_engine and workflow_engine.lower() == 'tekton':
```

Comment:
This all looks good.  Just wondering if we should precede this `if` block with `if not workflow_engine:` and raise a `ValueError` that it's required? This will clean up the `if/elif` statements.  Then add an `else` with another `ValueError` that an unsupported workflow engine was detected?